kernelci/build.py

```diff
@@ -802,6 +805,22 @@ def _kernel_config_enabled(self, config_name):
         cmd = 'grep -cq CONFIG_{}=y {}'.format(config_name, dot_config)
         return shell_cmd(cmd, True)
 
+    def _kernel_config_getkey(self, opt_name):
+        dot_config = os.path.join(self._output_path, '.config')
+        with open(dot_config, 'r') as fp:
+            for line in fp:
+                if line.startswith(opt_name):
+                    pcfg = line.strip().split('=')
+                    if len(pcfg) > 1 and pcfg[0] == opt_name:
+                        return pcfg[1]
```

Comment:
Note: This could even be done with a regex and groups to directly match and parse, but let's not block the PR on that ;)